_data/health.yml

```diff
@@ -230,6 +230,15 @@ websites:
     facebook: thriva
     email_address: support@thriva.co
 
+  - name: Virgin Pulse
+    url: https://www.virginpulse.com/
+    img: virginpulse.png
+    tfa:
+      - sms
+      - email
+    doc: https://virginpulse.zendesk.com/hc/en-us/articles/360033807991
+    exception: "SMS 2FA is only available for US and Canada phone numbers."
+
```

Comment:
Seems like it should be limited to U.S. use only.
![virginpulse](https://user-images.githubusercontent.com/3535780/118892127-b7d10700-b900-11eb-8011-da2e9ffa0fb9.png)

```suggestion
    regions:
      - us

```

Reply:
That's strange - when I visit the site, I get redirected to a UK version. There also seems to be regional sites for various other countries. e.g. https://www.virginpulse.com/de/

And they do seem to have various global contacts (https://www.virginpulse.com/en-gb/contact/): 
![image](https://user-images.githubusercontent.com/20560218/119011316-d8df3980-b98c-11eb-9bda-3eaf748bbba2.png)

The T&C's might not be fully accurate?

Reply:
I emailed them and asked. With no GDPR part in their privacy policy and no mention of any SCC I get the feeling they may not yet be cleared to operate in the EU, EEA, UK :thinking:

Reply:
Is this supposed to be our problem? We can hardly monitor for changes in privacy policies or the like and update our data accordingly.That would create a maintenance nightmare.

Reply:
I've had a look, and they do seem to mention GDPR and SCC in their [privacy policy](https://www.virginpulse.com/en-gb/privacy-notice/) now:
![image](https://user-images.githubusercontent.com/20560218/123520117-06fc1b80-d6a7-11eb-8476-3f7e7c9be491.png)

They also mention that they are available in 190+ countries on their [About Us](https://www.virginpulse.com/en-gb/about-us/) page:
![image](https://user-images.githubusercontent.com/20560218/123520172-53475b80-d6a7-11eb-819f-a364ee3099d1.png)

It seems that they do operate globally, so no need for a regions array.